src/Microsoft.IdentityModel.Tokens/SecurityAlgorithms.cs

```diff
@@ -31,6 +31,9 @@ public static class SecurityAlgorithms
         public const string Aes256KW = "A256KW";
         public const string RsaPKCS1 = "RSA1_5";
         public const string RsaOAEP = "RSA-OAEP";
+#if NET461 || NET462 || NET472 || NETSTANDARD2_0 || NET6_0_OR_GREATER
```

Comment:
Yes, thanks for pointing that out.  I think that the removal of 4.5.2 support happened just after I put in this PR.  I'll remove this since it should always be included now.